quiche/src/lib.rs

```diff
@@ -8023,7 +8113,9 @@ pub struct TransportParams {
     pub retry_source_connection_id: Option<ConnectionId<'static>>,
     /// DATAGRAM frame extension parameter, if any.
     pub max_datagram_frame_size: Option<u64>,
-    // pub preferred_address: ...,
+    /// Unknown peer transport parameters and values, if any.
+    pub unknown_params: UnknownTransportParameters, /* pub preferred_address:
+                                                     * ..., */
```

Comment:
I think cargo fmt might have tried to be clever here with the `preferred_address` . Can we move it to its own line underneath please?